snapshots/climate/2023-12-20/surface_temperature.gz.dvc

```diff
@@ -1,25 +1,27 @@
 meta:
   origin:
-    title_snapshot: ERA5-Land monthly averaged data from 1950 to present - Monthly
+    title_snapshot: ERA5 Monthly Averaged Data on Single Levels from 1940 to Present - Monthly
```

Comment:
no, it is actually 1940  --> https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels-monthly-means?tab=overview

Reply:
The charts and datasets we currently have in grapher start in 1950 (it's even written as the title of the datasets, e.g. [this one](https://owid.cloud/admin/datasets/6352)). If now they start in 1940, then you should change the metadata accordingly. Or are we removing the data for the first decade for some reason?

Reply:
I was originally mistakenly only extracting data for 1950 and above but I've fixed it now. I've changed the metadata fields in the datasets accordingly as well.

Reply:
Good, thanks for fixing that. Now that you are changing it, I think in general it would be better to not include the coverage (in this case 1940) in the titles, since that's something that could eventually change.

Reply:
ok done